### PR TITLE
Update Event “davos”

### PR DESCRIPTION
**Pull request by mfbgsz**
Automatically generated by Decap CMS

### CODE REVIEW

**Comment by mfbgsz** [comment]
@CharlyMartin I couldn't delete “davos-penthouse-25” image - can you please delete it when you can? Thank youu

File: src/content/events/davos.md [hunk 1]
```diff
@@ -13,12 +14,142 @@ location:
...
+        (seating up to 10 people) is available upon request. Located in WEF’s
+        Secure Zone, the venue requires a WEF Hotel Badge or other official WEF
+        event badge or credential for entry. Book a private meeting room:
+        events@fil.org "
+      external-link: https://cvent.me/xbZ2w2
+    - title: " With Honor Action Reception. With Special Guests: Members of the U.S.
+        Congressional Delegation. Hosted by Filecoin Foundation, With Honor
+        Action"
+      start-date: 2025-01-22T20:30:00.000Z
+      end-date: ""
```
Comment (CharlyMartin) [inline src/content/events/davos.md:36]: @mfbgsz `end-date` fields are not dates, so the build fails.

If there is no end date, leave it empty.

Thanks!

**Comment by CharlyMartin** [comment]
> @CharlyMartin I couldn't delete “davos-penthouse-25” image - can you please delete it when you can? Thank youu

@mfbgsz Done!

File: src/content/events/davos.md [hunk 2]
```diff
@@ -13,12 +14,145 @@ location:
...
   twitter:
     card: summary
-  description: Join FIL.org in Davos for high-level connections and conversations
-    on Web3, decentralization, and the future of the internet. Open daily
-    9am-5pm. Don’t miss this premier event!
-  title: Explore Web3 at Davos | FIL.org
+  description: Join the Filecoin community in Davos during the World Economic
+    Forum (Jan 20-23, 2025). Explore speaking engagements and events shaping the
+    future of the decentralized web.
+  title: FF at Davos 2025 | fil.org
```
Comment (CharlyMartin) [inline src/content/events/davos.md:157]: @mfbgsz The ` | fil.org` part is unnecessary, we automatically include ` | Filecoin Foundation` at the end of the SEO title.

```suggestion
  title: FF at Davos 2025
```

File: src/content/events/davos.md [hunk 3]
```diff
@@ -13,12 +14,145 @@ location:
...
+  title: Featured Events
+  kicker: Find us around Davos
+schedule:
+  title: Speaking Engagements
+  days:
+    - events:
+        - title: Marta Belcher at GBBC’s 8th Annual Blockchain Central Davos
+          location: Grandhotel Belvedere | Promenade 89, 7270 Davos, Switzerland
+          tag: Registration Needed
+          start: 2025-01-20T00:00:00.000Z
```
Comment (CharlyMartin) [inline src/content/events/davos.md:67]: This says midnight, maybe let's use noon or something generic if we don't know the time yet?

File: src/content/events/davos.md [hunk 4]
```diff
@@ -13,12 +14,145 @@ location:
...
+          end: 2025-01-22T00:00:00.000Z
+          url: https://www.gbbc.io/events/save-the-date-8th-annual-blockchain-central-davos
```
Comment (CharlyMartin) [inline src/content/events/davos.md:69]: This link says the event happens on the 22nd while being listed as the 20th on our side

File: src/content/events/davos.md (hunk 3 above)
Comment (mfbgsz) [reply src/content/events/davos.md:67]: I'm ok with that - people will not see this until closer to the date... ideally no time would be best, but ok if we need to put something!

File: src/content/events/davos.md (hunk 4 above)
Comment (mfbgsz) [reply src/content/events/davos.md:69]: their website is wrong 😂 it is Jan 20-22, but let's leave it for now on the 20th until we know exactly when she is speaking (date/time)

File: src/content/events/davos.md (hunk 2 above)
Comment (mfbgsz) [reply src/content/events/davos.md:157]: sounds good! commiting

**Pull request merged**